automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,468 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
+              val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName)
 
-            val workspaceSummaryPage = new WorkspaceSummaryPage(projectName, workspaceName)
-            go to workspaceSummaryPage
-            workspaceSummaryPage.awaitLoaded()
-            workspaceSummaryPage.ui.readError() should include(projectName)
-            workspaceSummaryPage.ui.readError() should include(workspaceName)
-            workspaceSummaryPage.ui.readError() should include("does not exist")
+              val cloneWorkspaceName = workspaceName + "_clone"
+              val cloneModal = summaryPage.ui.clickCloneButton()
+              cloneModal.ui.readLockedAuthDomainGroups() should contain(authDomainName)
+
+              register cleanUp {
+                api.workspaces.delete(projectName, cloneWorkspaceName)(AuthTokens.george)
+              }
+              cloneModal.cloneWorkspace(projectName, cloneWorkspaceName)
+              cloneModal.cloneWorkspaceWait()
+              summaryPage.cloneWorkspaceWait()
+
+              val cloneSummaryPage = new WorkspaceSummaryPage(projectName, cloneWorkspaceName).awaitLoaded()
+              cloneSummaryPage.ui.readWorkspaceName should be(cloneWorkspaceName)
+              cloneSummaryPage.ui.readAuthDomainGroups should include(authDomainName)
+            }
           }
         }
       }
 
-      "should not be visible" in withWebDriver { implicit driver =>
-        withGroup("AuthDomainSpec") { authDomainName =>
-          withWorkspace(projectName, "AuthDomainSpec_share", Some(authDomainName)) { workspaceName =>
-            val listPage = signIn(Config.Users.ron)
-            listPage.filter(workspaceName)
-            listPage.ui.hasWorkspace(projectName, workspaceName) shouldEqual false
+      "when the user is not inside of the group" - {
+        "when the workspace is shared with them" - {
+          "can be seen but is not accessible" in withWebDriver { implicit driver =>
+            withGroup("AuthDomainSpec") { authDomainName =>
+              withWorkspace(projectName, "AuthDomainSpec_reject", Set(authDomainName)) { workspaceName =>
+                api.workspaces.updateAcl(projectName, workspaceName, Config.Users.ron.email, WorkspaceAccessLevel.Reader)
+
+                val workspaceListPage = signIn(Config.Users.ron)
+                workspaceListPage.filter(workspaceName)
+                workspaceListPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
+
+                workspaceListPage.ui.clickWorkspaceInList(projectName, workspaceName)
+                // micro-sleep just long enough to let the app navigate elsewhere if it's going to, which it shouldn't in this case
+                Thread sleep 500
+                workspaceListPage.validateLocation()
+                // TODO: add assertions for the new "request access" modal
+              }
+            }
+          }
+          "when the user is a billing project owner" - {
```

Comment:
i'm replacing all of the billing project owner tests with these three tests:

when there is a multi-group auth domain and it is shared with a billing project owner who is not in all of those groups, it should be seen but not accessible

when there is a multi-group auth domain and it is shared with a billing project owner who is in all of those groups, it should be seen and accessible

when there is a multi-group auth domain and it is not shared with a billing project owner, it should be seen but not accessible

i think that covers all of the real cases. the single-group auth domain tests were just redundant